pkg/netrasp/sros.go

```diff
@@ -0,0 +1,97 @@
+package netrasp
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"strings"
+)
+
+// SROS is the Netrasp driver for Nokia SR OS (MD-CLI).
+type sros struct {
+	Connection connection
+}
+
+// Close connection to device.
+func (i sros) Close(ctx context.Context) error {
```

Comment:
Perhaps change from func (i sros) to func (s sros)?

Reply:
done in 133b5d7